### PR TITLE
fix conflict logical in properties namesapce.

### PR DESCRIPTION
**Pull request by JaredTan95**
Verify whether to repeat in text mode, when adding `properties` config.

### CODE REVIEW

File: apollo-portal/src/main/java/com/ctrip/framework/apollo/portal/component/txtresolver/PropertyResolver.java [hunk 1]
```diff
@@ -89,6 +89,12 @@ private boolean isHasRepeatKey(String[] newItems) {
       lineCounter++;
     }
 
+    keys.forEach(item -> {
```
Comment (nobodyiam) [inline apollo-portal/src/main/java/com/ctrip/framework/apollo/portal/component/txtresolver/PropertyResolver.java]: how about changing line 84 to `keys.add(kv[0].toLowerCase());`?

Comment (JaredTan95) [reply apollo-portal/src/main/java/com/ctrip/framework/apollo/portal/component/txtresolver/PropertyResolver.java]: In my opinion, this way would be more flexible. It won't limit the user to use all lowercase key in properties. :-P

Comment (nobodyiam) [reply apollo-portal/src/main/java/com/ctrip/framework/apollo/portal/component/txtresolver/PropertyResolver.java]: by changing the key to lowercase in line 84 won't affect the actual properties, it's only a way to check whether there are case-insensitive duplicated keys...

Comment (JaredTan95) [reply apollo-portal/src/main/java/com/ctrip/framework/apollo/portal/component/txtresolver/PropertyResolver.java]: Aha, I got this point. :-D

**Pull request merged**